packages/blocks/src/database-block/database-block.ts

```diff
@@ -289,18 +298,25 @@ export class DatabaseBlockComponent extends NonShadowLitElement {
     });
   };
 
+  private _initTitleVEditor() {
+    this._vEditor = new VEditor(this.model.title.yText);
+    this._vEditor.mount(this._container);
+    this._vEditor.focusEnd();
+    this._vEditor.setReadonly(this.model.page.readonly);
+
+    // for title placeholder
+    this.model.title.yText.observe(() => {
```

Comment:
@Flrande shall we expose the `yText` content here?

Reply:
Unless we choose to put the text-adapter into virgo. 🤔